public/static/docs/command-reference/diff.md

```diff
@@ -19,9 +19,9 @@ positional arguments:
 
 ## Description
 
-Given two Git commit references (commit hash, branch or tag name, etc) `a_ref`
-and `b_ref`, this command shows a comparative summary of basic statistics: how
-many files were deleted/changed, and the file size differences.
+Given two Git commit references (commit SHA hash, branch or tag name, etc)
```

Comment:
commit hash -> commit SHA hash
> In the context of Git commit SHA-1 hashes

Reply:
Per private chat with Ivan we're going for just "commit hash" here. Just being careful that the phrase doesn't happen in the same context as file hash, to avoid confusion.

Reply:
OK, reverted in 67bc4555.